timeseries/src/autogluon/timeseries/models/autogluon_tabular/mlforecast.py

```diff
@@ -87,6 +94,26 @@ def __init__(
         self._avg_residuals_std: Optional[float] = None
         self._train_target_median: Optional[float] = None
 
+    @property
+    def tabular_predictor_path(self) -> str:
+        return os.path.join(self.path, "tabular_predictor")
+
+    def save(self, path: str = None, verbose: bool = True) -> str:
+        trained_models = self._mlf.models_
+        self._mlf.models_ = None
+        save_path = super().save(path=path, verbose=verbose)
+        self._mlf.models_ = trained_models
+        return save_path
+
+    @classmethod
+    def load(
+        cls, path: str, reset_paths: bool = True, load_oof: bool = False, verbose: bool = True
+    ) -> "AbstractTimeSeriesModel":
+        model = super().load(path=path, reset_paths=reset_paths, load_oof=load_oof, verbose=verbose)
+        # Assumes that model was fit before saving and tabular_predictor_path exists
+        model._mlf.models_ = {"mean": TabularEstimator.load(model.tabular_predictor_path)}
```

Comment:
where are the individual quantiles handled?

Reply:
The quantiles are added differently for DirectTabular & RecursiveTabular; the actual logic is stored in methods like `_postprocess_predictions` and `_add_gaussian_quantiles`.

Always calling the model `mean` just ensures that for RecursiveTabular the output of `MLForecast.predict` is a DataFrame with column `mean` containing the point forecast.